actions/reservationActions.tsx

```diff
@@ -0,0 +1,61 @@
+"use server";
+
+import { db } from "../database/database";
+import { reservations } from "../database/schemas";
+import { eq, gte } from "drizzle-orm";
+
+export const retrieveReservations = async (fromDate?: string): Promise<{ reservations: any[] | null; error: string | null }> => {
```

Comment:
`any[]` causes inconsistencies from the output, use `any` instead, since the return on line `29` is a typed as a complex type from `drizzle`.